mongodbatlas/fw_resource_mongodbatlas_encryption_at_rest_test.go

```diff
@@ -116,28 +117,42 @@ func TestAccAdvRSEncryptionAtRest_basicAWS(t *testing.T) {
 			SecretAccessKey:     os.Getenv("AWS_SECRET_ACCESS_KEY_UPDATED"),
 			CustomerMasterKeyID: os.Getenv("AWS_CUSTOMER_MASTER_KEY_ID_UPDATED"),
 			Region:              os.Getenv("AWS_REGION_UPDATED"),
+			RoleID:              os.Getenv("AWS_ROLE_ID"),
 		}
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:          func() { testAccPreCheck(t); testCheckAwsEnv(t) },
-		ProviderFactories: testAccProviderFactories,
-		CheckDestroy:      testAccCheckMongoDBAtlasEncryptionAtRestDestroy,
+	resource.Test(t, resource.TestCase{
```

Comment:
why this is not Parallel anymore?

Reply:
This was updated because currently these tests do not run on Github pipeline, they can only be run locally. Also, they currently use the same project to manipulate the same resource so these can't be run in parallel even locally. 

The project isn't created inside the test configs as the same project is used for calling update configs as well.

These can be parallel again once we have Azure/GCP credentials setup as well and all tests actually run in actions. Happy to discuss more on this.